cpp/include/cuvs/neighbors/cagra.hpp

```diff
@@ -104,7 +108,8 @@ struct index_params : cuvs::neighbors::index_params {
    */
   std::variant<std::monostate,
                graph_build_params::ivf_pq_params,
-               graph_build_params::nn_descent_params>
+               graph_build_params::nn_descent_params,
```

Comment:
We have explicit docs for each of these arguments. I understand the iterative search params are experimental to start, but can we at least add them to the docs?

Reply:
Thanks, I will add comments about iterative_search_params here.